### ISSUE TITLE
[Enhancement]: Accessibility Issue on Cart Page: Focus Indicator on Pay buttons does not meet contrast ratios

### ISSUE THREAD

**Issue by raquel-emberly** [woothemes/woocommerce#60721]
### Describe the solution you'd like

On the cart page when a shift in color is the only visual indication of focus or selection, that color shift must meet the minimum color contrast required by the WCAG. The combination of green (#00C767) and green (#00D66F) only has a contrast ratio of 1.15:1.

Ensure the color in the focused or selected state is at least a 3:1 contrast ratio with the color in the unfocused or unselected state, while maintaining a 3:1 contrast ratio with its background. Additionally, if text is involved, the text must maintain a 4.5:1 contrast ratio with its background in either state.

If this is not possible, an additional visual cue must be added to indicate the focused or selected state, such as text that has an underline or becomes bolder (700 or greater) or by adding a border.

<img src="https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/ba643ab0-91e9-4289-8e86-55ada92cdf10/72514e6a-6826-40d8-976e-021739fda962?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi9iYTY0M2FiMC05MWU5LTQyODktOGU4Ni01NWFkYTkyY2RmMTAvNzI1MTRlNmEtNjgyNi00MGQ4LTk3NmUtMDIxNzM5ZmRhOTYyIiwiaWF0IjoxNzczMzIzODQ5LCJleHAiOjE4MDQ4OTQ0MDl9.YACy9oNEwWn750v_hO6rnYJPStRvksEvS7YIgqHgkbY " alt="Image" width="1865" data-linear-height="750" />

Affected users<br>Visual

Guidelines<br>[1.4.1 (WCAG 2.0 A)Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.](<https://www.w3.org/WAI/WCAG22/Understanding/use-of-color>), [1.4.11 (WCAG 2.1 AA)](<https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast>)

### Describe alternatives you've considered

*No response*

### Additional context

*No response*

**Comment by nerrad** [comment]
Hi there, what "shift of color" are you referring to? It's not clear from your screenshot. I'm also not seeing either of the hex values you referenced in the codebase (`#00C767`, or `#00D66F`) - it's possible those were set either by the theme, or within the editor for the block.

**Comment by raquel-emberly** [comment]
When navigating via only a keyboard, tab through the page content. When you tab to payment button options - Apple Pay, Google Pay, Link Pay - there is not enough visibility shift when these buttons have focus. 

https://github.com/user-attachments/assets/5ea949a5-fc39-4bfa-b511-c889f94fcc69

**Comment by nerrad** [comment]
Thanks! That's helpful. This *might* be payment method specific. What payment plugin are you using that provide these express payment method options?

**Comment by raquel-emberly** [comment]
We are using the **WooCommerce Stripe Gateway** plugin with settings _Express checkouts_  enabling Apple Pay / Google Pay and Link by Stripe

**Comment by nerrad** [comment]
Thanks again. One more thing, I see you've submitted several accessibility related issues. One thing that will be helpful is to make sure the WooCommerce and WordPress version you are testing against are clear in the issue. If you're not testing against the latest versions, it's important to do so to verify it's still present in a recent release.

**Comment by raquel-emberly** [comment]
Yes correct, we are running the latest version of Wordpress, WooCommerce, PHP and all related plugins
